Initial runthrough for clarity (#109)
nfidd/nfidd: sessions/delay-distributions.qmd

## Simulating delayed epidemiological data

We will start this session by working with a simulated data set of infections from a disease that has caused an outbreak which subsequently ended.
In our example outbreak, there were 649 infections.
The outbreak lasted around 90 days after the first infection, with new infections peaking roughly around day 40.

::: callout-note
For now we will not concern ourselves with the model used to generate the epidemic.
This represents a typical situation in the real world, where we may have a model of how an infection has spread, but we don't know necessary know how well this corresponds to what really happened.
:::

We will later deal with modelling the infectious process.
For now, we will focus on modelling how infections get reported in data - the *observation* process.

> This has three issues:

Reviewer: ```suggestion
Using infectious disease data for analysis comes with three common challenges:
```

1.  We don't normally observe infections directly, but their *outcomes* as symptomatic cases, hospitalisations or other realisations.

> 2.  These observations are *incomplete* (e.g. because not every infection leads to hospitalisation, and so focusing on hospitalisations may leave infections unobserved)

Reviewer: ```suggestion
2.  These observations are *incomplete* (e.g. not every infection leads to hospitalisation, and so focusing on hospitalisations will leave infections unobserved)
```
